### PR TITLE
Added Professor Class

### PR DESCRIPTION
**Pull request by YuvalDeshe**
Added the Professor class to be used in webscraping; #21

### CODE REVIEW

**Comment by Jeffrey-000** [comment]
some issues:
-create class constructors
-change the class property visibility to private; currently is package protected
-create necessary getters and setters
-specifically line 11 note: setting the values of the array should be done by a constructor and after the variables have be initialized

**Comment by YuvalDeshe** [comment]
The commit 07510e0:
 - Added 3 constructors.
 - Made all variables within the class private.
 - Created a getter and a setter method for each variable (two setters for ratingDistribution).
 - Added documentation for each method.

File: backend/webscraping/Professor.java [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+     * @param numFiveStar The number of five-star ratings given to this professor.
+     * @param numRatings The number of ratings that the professor has.
+     * @param tags The tags associated with the professor.
+     * @param department The department this professor works in.
+     */
+    @Override
+    public Professor(String name, double quality, double wouldTakeAgainPercent, double difficulty, 
+                     int numOneStar, int numTwoStar, int numThreeStar, int numFourStar, 
+                     int numFiveStar, int numRatings, String[] tags, String department) {
+        name = this.name;
```
Comment (Jeffrey-000) [inline backend/webscraping/Professor.java:62]: the variables are in the wrong order
should be --> this.name = name;

**Review by Jeffrey-000 — CHANGES_REQUESTED** [review]

**Comment by YuvalDeshe** [comment]
The commit 40de972 fixed the order of the variables.

**Comment by YuvalDeshe** [comment]
The commits 1de790b and b53cf0a moved the file to the correct directory according to the new file structure and added Professor.java to the `edu.uga.devdogs.professor_rating.webscraping` package.

**Pull request merged**